Source/CarthageKit/Xcode.swift

```diff
@@ -172,6 +172,11 @@ public struct BuildArguments {
 		// Disable code signing requirement for all builds
 		// Frameworks get signed in the copy-frameworks action
 		args += [ "CODE_SIGNING_REQUIRED=NO", "CODE_SIGN_IDENTITY=" ]
+		
+		// Allow to force unsanitized environment
```

Comment:
Could you expand this comment to explain what `XCODEBUILD_USE_SANITIZED_ENVIRONMENT` is exactly?